src/pages/BaseWizardPage.js

```diff
@@ -73,7 +74,17 @@ class BaseWizardPage extends Component {
     return false;
   }
 
-  renderNavButtons() {
+  closeUpdateProcess(e) {
```

Comment:
These functions that are specific to the update do not belong in this class, which is a base class for any page that appear in the wizard.

Reply:
All the update pages reused most of the components from the BaseWizardPage,  Do you mean I need to have a another say BaseUpdateWizardPage which extends from BaseWizardPage to have closeUpdateProcess and cancelUpdateProcess moved there?

Reply:
Yes, exactly. The base class should only contain things that are common to all pages